style.css

```diff
@@ -0,0 +1,396 @@
+* {
+  margin: 0;
+  padding: 0;
+  font-family: 'Lato', sans-serif;
+}
+
+.primary {
+  color: #272a31;
+}
+
+.secondary-color {
+  color: #ec5242;
+}
+
+.accent-color {
+  color: #d3d3d3;
+}
+
+.primary-bg {
+  background: #272a31;
+}
+
+.secondary-bg {
+  background: #ec5242;
+}
+
+.accent-bg {
+  background: #d3d3d3;
+}
+
+.nav-link.active {
+  color: #ec5242 !important;
+}
+
+@media (min-width: 320px) {
+  .top-header {
+    background: #272a31;
+  }
+
+  .topHeader {
+    height: 40px;
+    gap: 12px;
+    justify-content: center;
+  }
+
+  .topHeader a {
+    color: #fff;
+    font-size: 0.8em;
+  }
+
+  .navbar {
+    position: absolute;
+    background: transparent !important;
+  }
+
+  .navbar-brand img {
+    width: 80px;
+  }
+
+  .nav-link {
+    color: #272a31 !important;
+    font-size: 1em;
+  }
+
+  .navbar-toggler {
+    border: none;
+    padding: 0.25rem 0;
+  }
+
+  #top-banner {
+    background: url(images/mobile-top-banner.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-position: center center;
+    width: 100%;
+    height: 750px;
+  }
+
+  .event-headline {
+    height: inherit;
+    gap: 15px;
+  }
+
+  .about-headline {
+    height: inherit;
+    gap: 25px;
+  }
+
+  .event-headline-main {
+    /* mix-blend-mode: color-burn; */
+    font-size: 3em;
+    line-height: 52px;
+    font-weight: 900;
+    color: #ec5242;
+  }
+
+  .event-summary {
+    padding: 1em 2em;
+    border: 2px solid #fff;
+    background: #d3d3d3;
+  }
+
+  .event-summary p {
+    color: #272a31;
+  }
+
+  #usp {
+    background: #272a31;
+    padding: 2em 0;
+    height: auto;
+    gap: 15px;
+  }
+
+  .title-line {
+    border: 2px solid #ec5242;
+    width: 50px;
+  }
+
+  .about-title-line {
+    border: 2px solid #ec524261;
+    width: 50px;
+  }
+
+  .usp-list {
+    width: 100%;
+    height: 150px;
+    background: #d3d3d34f;
+    margin-bottom: 10px;
+    gap: 10px;
+    transition: all 1s;
+  }
+
+  .usp-list:hover {
+    border: 3px solid #fff;
+  }
+
+  .usp-list i.fa {
+    font-size: 2em;
+  }
+
+  .usp-list h5 {
+    font-size: 1em;
+    font-weight: 900;
+  }
+
+  .usp-list p {
+    font-size: 0.8em;
+  }
+
+  #featured-speakers {
+    padding: 2em 0;
+  }
+
+  #featured-speakers .container {
+    gap: 15px;
+  }
+
+  #featuredSpeakers {
+    display: block;
+  }
+
+  .featured-speakers-2 {
+    gap: 10px;
+  }
+
+  .featured-speakers-1 {
+    gap: 8px;
+    text-align: center;
+  }
+
```

Comment:
- Hey, great job adding the speaker section using javascript as pointed out by the previous reviewer, however, there are some changes you still need to make to improve your design on each card on the mobile version. see the suggestions below and kindly implement them as much as you can:

   - You design on each card looks totally different from the main design. see the images below to improve on your design
   - Align the contents on each card correctly following what you see on the main design.
   - Use the correct font styles on text, size.
   - Add the missing horizontal gray line.
   - Use the correct gap between the image and the text content. etc.
   
 *Your design* 
![image](https://github.com/sulaimonaa/music-concert/assets/59397159/8e757ed8-efb3-4397-af6c-d3fd26a8d919)


*Main design*
![image](https://github.com/sulaimonaa/music-concert/assets/59397159/20642395-fd45-4fa2-8db0-1eb1f506a0ab)